pkg/agent/agent_test.go

```diff
@@ -151,3 +151,21 @@ func TestPersistRoundNum(t *testing.T) {
 	// expectations above).
 	persistRoundNum(roundNum, mockOVSBridgeClient, 0, maxRetries)
 }
+
+func TestGetRoundInfo(t *testing.T) {
+	controller := mock.NewController(t)
+	defer controller.Finish()
+	mockOVSBridgeClient := ovsconfigtest.NewMockOVSBridgeClient(controller)
+
+	mockOVSBridgeClient.EXPECT().GetExternalIDs().Return(nil, ovsconfig.NewTransactionError(fmt.Errorf("Failed to get external IDs"), true))
+	roundInfo := getRoundInfo(mockOVSBridgeClient)
+	if roundInfo.RoundNum != 1 {
+		t.Errorf("Failed to get expected round info")
+	}
+	externalIDs := make(map[string]string)
+	mockOVSBridgeClient.EXPECT().GetExternalIDs().Return(externalIDs, nil)
+	roundInfo = getRoundInfo(mockOVSBridgeClient)
+	if roundInfo.RoundNum != 1 {
+		t.Errorf("Failed to get expected round info")
```

Comment:
maybe also add the expected and the actual value in the errorf.. general comment for all newly added tests